README.md

```diff
@@ -3,45 +3,64 @@ Web3j Open API
 
 [![Build Status](https://travis-ci.org/web3j/web3j-openapi.svg?branch=master)](https://travis-ci.org/web3j/web3j-openapi)
 
-## Modules
-
- * openapi: JAX-RS, Web3j Core (only interfaces), Swagger annotations
- * core: openapi, Jersey Core, Swager UI
- * server: core, Jersey Server, Jetty, integration tests
- * codegen: Mustache, Kotlinpoet?, openapi (Based on CorDappGenerator for Gradle project)
- * client: core, Jersey Client
- * console: Command-line tools command `epirus-openapi`, sub-commands `generate`, `run`
- * helloworld:
-   * openapi
-   * server: web3j-openapi-server (org.web3j.openapi.server.MainKt or generated main)
+Web3j-OpenAPI is an OpenAPI generator from solidity smart contracts. 
+
+## Description 
+Web3j-OpenAPI provides a way to interact with the Ethereum blockchain via simple and intuitive HTTP requests, abstracting the coding layer. These interactions can be done using plain HTTP requests or via the Swagger-UI, which is generated with every project.
+
+The workflow can be summed in the following steps:
+- Writing a solidity smart contract
+- Generating the corresponding OpenAPI project using Web3j-OpenAPI
+- Running the generated project
+- Sending HTTP requests using Curls or Swagger-UI
+
+### Why an OpenAPI generator
+As stated above, the Web3j-OpenAPI generator generates an OpenAPI project from a smart contract. Thus, providing a way to interact with the Etheruem blockchain using HTTP requests.
+
+Such generator is beneficial in the following ways:
+#### Making it easier to interact with the Ethereum blockchain:
+Interacting with the Ethereum blockchain before required knowing a programming language besides solidity, and then hardcoding the desired logic. This, makes it hard for people wishing to get involved in the smart contracts world and adds extra technical requirements in addition to knowing the smart contracts language.
+#### Interact with smart contracts without code:
+Being able to generate an OpenAPI project from a smart contract and interacting with it using HTTP requests, eliminates the need to code any interactions and being able to send them to the blockchain. Thus, it is legitimate to say that the Web3j-OpenAPI lets you interact with the Ethereum blockchain using no code, besides solidity.
+
 
 ## Use cases
+An OpenAPI project can be generated using the following command:
 
 ```ssh
-$ epirus generate openapi Greeter.sol -p com.helloworld -o /home/rachid/helloworld
+$ epirus generate openapi -p com.helloworld -o . --abi helloworld.abi --bin helloworld.bin --name helloworld
```

Comment:
this part of the documentation needs to go into epirus-cli. 

you can link to epirus-cli here instead

Reply:
what about now ?